### PR TITLE
Feature/Notification and 

### PR DESCRIPTION
**Pull request by sura1542**
Finish All Notification and Test UI
Pls Check and Approve

### CODE REVIEW

File: lib/App.dart [hunk 1]
```diff
@@ -3,14 +3,14 @@ import 'package:flutter/material.dart';
...
 import 'package:jitd_client/src/screens/autheentication/SignIn.dart';
 
-import 'package:jitd_client/src/screens/autheentication/SignUp.dart';
-import 'package:jitd_client/src/screens/post/ViewAllPost.dart';
-import 'package:jitd_client/src/screens/tutorials/TutorialPage1.dart';
-import 'package:jitd_client/src/screens/tutorials/TutorialPage2.dart';
+// import 'package:jitd_client/src/screens/autheentication/SignUp.dart';
+// import 'package:jitd_client/src/screens/post/ViewAllPost.dart';
+// import 'package:jitd_client/src/screens/tutorials/TutorialPage1.dart';
+// import 'package:jitd_client/src/screens/tutorials/TutorialPage2.dart';
```
Comment (thunyathep) [inline lib/App.dart:13]: why comment

File: lib/src/screens/Notification/Noti1.dart [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      //         decoration:
+      //             const BoxDecoration(color: primaryColorSubtle, boxShadow: [
+      //           BoxShadow(
+      //             blurRadius: 4,
+      //             color: Color.fromRGBO(0, 0, 0, 0.25),
+      //             offset: Offset(0, 4),
+      //           )
+      //         ]),
+      //         child: Column(
+      //           mainAxisAlignment: MainAxisAlignment.spaceBetween,
```
Comment (aofattaporn) [inline lib/src/screens/Notification/Noti1.dart:39]: pls remove all comment

File: lib/src/screens/Notification/Noti_test.dart [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+// MyApp is a StatefulWidget. This allows updating the state of the
+// widget when an item is removed.
+// class Noti_test extends StatefulWidget {
+//   const Noti_test({Key? key}) : super(key: key);
+//
+//   @override
+//   Noti_testState createState() {
+//     return Noti_testState();
+//   }
+// }
```
Comment (aofattaporn) [inline lib/src/screens/Notification/Noti_test.dart:16]: pls remove this

File: lib/src/screens/NotificationPage.dart [hunk 4]
```diff
@@ -1,27 +1,242 @@
...
+// import 'package:jitd_client/src/blocs/authentication/authen_state.dart';
+// import 'package:jitd_client/src/screens/post/ViewAllPost.dart';
+// import 'package:shimmer/shimmer.dart';
+//
+// import '../blocs/post/post_bloc.dart';
+// import '../blocs/post/post_state.dart';
+// void main() {
+//   runApp( NotificationPage());
+// }
+
```
Comment (aofattaporn) [inline lib/src/screens/NotificationPage.dart]: and this

**Review by aofattaporn — APPROVED** [review]

File: lib/App.dart (hunk 1 above)
Comment (thunyathep) [inline lib/App.dart]: remove this

File: lib/App.dart [hunk 5]
```diff
@@ -3,14 +3,14 @@ import 'package:flutter/material.dart';
 
 import 'package:jitd_client/src/BottomNavigationWidget.dart';
 import 'package:firebase_auth/firebase_auth.dart';
-import 'package:jitd_client/src/screens/HomePage.dart';
-import 'package:jitd_client/src/screens/ProfilePage.dart';
+// import 'package:jitd_client/src/screens/HomePage.dart';
+// import 'package:jitd_client/src/screens/ProfilePage.dart';
```
Comment (thunyathep) [inline lib/App.dart]: remove

File: lib/src/screens/Notification/Noti1.dart [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+      //                     icon: Image.asset(
+      //                       'assets/images/messenger.png',
+      //                       color: Colors.white,
+      //                     ),
+      //                     onPressed: () {},
+      //                   ),
+      //                 ),
+      //               ],
+      //             ),
+      //SizedBox(height: 100),
```
Comment (Balloii) [inline lib/src/screens/Notification/Noti1.dart:87]: Remove all the comment>

**Review by Balloii — APPROVED** [review]

**Review by thunyathep — APPROVED** [review]

**Pull request merged**